docker-compose.yaml

```diff
@@ -67,7 +67,7 @@ services:
       options:
         max-size: "20m"
     ports:
-      - "9944:9944"
+      - "127.0.0.1:9944:9944"
```

Comment:
The same should be for `boot_node` service.

Reply:
done